extensions/blocks/pinterest/pinterest.php

```diff
@@ -15,6 +15,44 @@
 const FEATURE_NAME = 'pinterest';
 const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
 const URL_PATTERN  = '#^https?://(?:www\.)?(?:[a-z]{2}\.)?pinterest\.[a-z.]+/pin/(?P<pin_id>[^/]+)/?#i'; // Taken from AMP plugin, originally from Jetpack.
+// This is the validate Pinterest URLs, converted from URL_REGEX in wp-content/mu-plugins/jetpack/extensions/blocks/pinterest/index.js.
```

Comment:
I'd recommend keeping that path relevant for both environments:

```suggestion
// This is the validate Pinterest URLs, converted from URL_REGEX in extensions/blocks/pinterest/index.js.
```